DataFormats/L1TMuon/interface/CPPFDigi.h

```diff
@@ -0,0 +1,77 @@
+// Class for RPC trigger primitives sent from CPPF to EMTF
+// Author Alejandro Segura -- Universidad de los Andes
+
+#ifndef DataFormats_L1TMuon_CPPFDigi_h
+#define DataFormats_L1TMuon_CPPFDigi_h
+
+#include "DataFormats/MuonDetId/interface/RPCDetId.h"
+#include <vector>
+
+namespace l1t {
+  
+  class CPPFDigi{
+    
+  public:
+
+    CPPFDigi() :
+    bx(-99), phi_int(-99), theta_int(-99), valid(-99),
+      board(-99), channel(-99), emtf_sector(-99), emtf_link(-99), 
+      first_strip(-99), cluster_size(-99), phi_glob(-99), theta_glob(-99)
+      {};
+    
+    explicit CPPFDigi( const RPCDetId& _rpcId, int _bx, int _phi_int, int _theta_int, int _valid,
```

Comment:
Just forgot: CMS code rules [1] requires that ```Do not use “_” as first character``` (it applies here to all arguments in the constructors of this class)

[1] https://cms-cpt-software.web.cern.ch/cms-cpt-software/General/CodingAndStyleRules.pdf

Reply:
Right. fixing it including the one above RPCId and others.